src/test/java/org/folio/dao/DataImportQueueDaoTest.java

```diff
@@ -168,6 +164,80 @@ public void shouldGetQueueItemByIdFailure(TestContext context) {
       .onSuccess(er -> context.fail("Provided ID should not exist."));
   }
 
+  @Test
+  public void shouldGetWaitingQueueItems(TestContext context) {
+    // given
+    doAnswer((InvocationOnMock invocation) -> {
+        Promise<RowSet<Row>> promise = invocation.getArgument(2);
+        promise.complete(new LocalRowSet(3));
+        ArrayTuple tuple = invocation.getArgument(1);
+        assertThat(tuple.size(), is(1));
+        assertThat(tuple.getBoolean(0), is(false));
+        return null;
+      })
+      .when(postgresClient)
+      .select(
+        anyString(),
+        any(Tuple.class),
+        ArgumentMatchers.<Promise<RowSet<Row>>>any()
+      );
+
+    // when
+    queueItemDaoImpl
+      .getAllWaitingQueueItems()
+      // then
+      .onComplete(
+        context.asyncAssertSuccess(x -> {
+          verify(postgresClient, times(1))
+            .select(
+              startsWith(
+                "SELECT * FROM data_import_global.queue_items WHERE processing = "
```

Comment:
should this be extended to check if the boolean actually equals true or false?  otherwise there is no difference between this test case and the one after it?